docs/content/config/security/understanding-the-ports.md

```diff
@@ -16,7 +16,7 @@ Prefer ports with Implicit [TLS][wikipedia-tls] ports, they're more secure than
 | IMAP4                    | 143                      | 993             | Retrieval            | Yes                |
 
 1. A connection _may_ be secured over TLS when both ends support `STARTTLS`. On ports 110, 143 and 587, DMS will reject a connection that cannot be secured. Port 25 is [required][ref-port25-mandatory] to support insecure connections.
-2. Receives email, DMS additionally filters for spam and viruses. For submitting email to the server to be sent to third-parties, you should prefer the _submission_ ports (465, 587) - which require authentication. Unless a relay host is configured (eg: SendGrid), outgoing email will leave the server via port 25 (_thus outbound traffic must not be blocked by your provider or firewall_).
+2. Receives email, DMS additionally filters for spam and viruses. For submitting email to the server to be sent to third-parties, you need to use the _submission_ ports (465, 587) - which require authentication. Unless a relay host is configured (eg: SendGrid), outgoing email will leave the server via port 25 (_thus outbound traffic must not be blocked by your provider or firewall_).
```

Comment:
While this one is not specific to authenticated submission.

You may have another container / server send mail to DMS on port 25 to relay externally elsewhere. Provided it's trusted via something like `PERMIT_DOCKER`, it will be able to send that mail.

Presently `SPOOF_PROTECTION=1` doesn't affect port 25, so such a scenario could send as anyone on port 25 when trusted even with this ENV enabled. Our method used is only related to authenticated SASL users being prevented from sending as anyone but themselves though, whereas a different Postfix option would be better for port 25 that prevents sending as a known SASL user (_without authenticating, which would be required over 465 or 587 instead_).

I'd still advise using authentication to send mail out, vs trusting an IP address where any process could from that source could use DMS as an open relay (_from that trusted source at least_).

Reply:
Got it! Thank you for the clarification 😊  
This change has been reverted.